pkg/network/kafka/kafka_stats.go

```diff
@@ -0,0 +1,73 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package kafka
+
+import "github.com/DataDog/datadog-agent/pkg/process/util"
+
+const (
+	ProduceAPIKey = 0
+	FetchAPIKey   = 1
+)
+
+// KeyTuple represents the network tuple for a group of Kafka transactions
+type KeyTuple struct {
```

Comment:
This is indentical to `http.KeyTuple`, so maybe a good candidate for a common (`usm`) package?

Reply:
> indeed, but does not worth the effort here (forked branch)
make sense :+1: